libdnf/nsvcap.cpp

```diff
@@ -31,7 +31,7 @@ namespace libdnf {
 #define MODULE_ARCH MODULE_NAME
 #define MODULE_PROFILE MODULE_NAME
 
-constexpr const char * module_form_regex[] = {
+constexpr const char * nsvcap_regex[] = {
```

Comment:
I think this should be named `nsvcapRegex`.

Reply:
@mhatina And what about "NSVCAP_REGEX", it is constant array?

Reply:
It is array of constants and also `constexpr` itself, I would leave it in camel case. But it's good point to add to guidelines doc and discuss it more.